src/EPR.CommonDataService.Core.UnitTests/Extensions/StoredProcedureExtensionsTests.cs

```diff
@@ -75,23 +75,23 @@ public void ToProcParams_ShouldHandleNullDecisionsDelta()
         var result = request.ToProcParams();
 
         // Assert
-        var descisionDelta = result.Single(x => x.ParameterName == "@DecisionsDelta");
-        descisionDelta.Value.Should().Be("[]");
+        var decisionDelta = result.Single(x => x.ParameterName == "@DecisionsDelta");
+        decisionDelta.Value.Should().Be("[]");
     }
 
     [TestMethod]
     public void ToProcParams_ShouldHandleEmptyArrayDecisionsDelta()
     {
         // Arrange
         var request = _fixture.Build<SubmissionsSummariesRequest<RegulatorPomDecision>>()
-            .With(x => x.DecisionsDelta, new RegulatorPomDecision[]{})
+            .With(x => x.DecisionsDelta, Array.Empty<RegulatorPomDecision>())
             .Create();
 
         // Act
         var result = request.ToProcParams();
 
         // Assert
-        var descisionDelta = result.Single(x => x.ParameterName == "@DecisionsDelta");
-        descisionDelta.Value.Should().Be("[]");
+        var decisionDelta = result.Single(x => x.ParameterName == "@DecisionsDelta");

```

Comment:
This should probably be SingleOrDefault to ensure the unit test doesn't error, with an optional `Should().NotBeNull()`